contracts/src/everlong/EverlongERC4626.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+// import { IEverlongERC4626 } from "../interfaces/IEverlongERC4626.sol";
+import { ERC4626 } from "solady/tokens/ERC4626.sol";
+
+contract EverlongERC4626 is ERC4626 {
+    /// @notice Whether virtual shares will be used to mitigate the inflation attack.
+    bool public constant useVirtualShares = true;
+
+    /// @notice Used to reduce the feasibility of an inflation attack.
+    uint8 public constant decimalsOffset = 3;
```

Comment:
Some justification for why it's 3 would be good. I understand that we'll need to test this to find the real value, so you can just leave a TODO.

Reply:
Totally chose this at random, will mark as TODO